schema.json

```diff
@@ -1187,25 +1266,29 @@
       },
       {
         "type": "checkbox",
-        "label": "Show &quot;Shop by Price&quot;",
+        "label": "Show &quot;Shop by Price&quot; in filters",
         "force_reload": true,
         "id": "shop_by_price_visibility"
       },
       {
         "type": "color",
-        "label": "Wishlist dropdown background color",
+        "label": "Wishlist dropdown background",
         "id": "dropdown--wishList-backgroundColor"
       },
+      {
+         "type": "checkbox",
+         "label": "Show product reviews",
+         "force_reload": true,
+         "id": "show_product_reviews_tabs"
+      },
       {
         "type": "select",
-        "label": "Number of Related Products",
-        "id": "productpage_related_products_count",
+        "label": "Number of product reviews",
+        "id": "productpage_reviews_count",
+        "reference": "show_product_reviews_tabs",
+        "reference_default": false,
         "force_reload": true,
         "options": [
-          {
-            "value": 0,
-            "label": "Disable"
-          },
```

Comment:
Currently there are two options to turn off "Number of product reviews." Either by marking the checkbox (show product reviews tabs) or selecting "Disable." We have gotten rid of the "Disable" value and set a reference to show_product_reviews_tab to avoid redundancy.


**Original Schema setup**
![screen shot 2018-08-21 at 8 29 56 am](https://user-images.githubusercontent.com/33278039/44412460-d6223200-a51d-11e8-9643-d3bc60dd5e40.png)


**Updated**
![screen shot 2018-08-21 at 8 43 22 am](https://user-images.githubusercontent.com/33278039/44412649-4f218980-a51e-11e8-9b9c-5f464c9f8e5e.png)

Reply:
@junedkazi @BC-EChristensen @Ubersmake